### PR TITLE
Several features

### PR DESCRIPTION
**Pull request by dframirez80**
New features:
- Change button texts.
- Change device selection label text.
- Change text when camera does not start automatically
- Select camera to start automatically. This feature is due to the use in NET MAUI Blazor the android device has several rear cameras and the camera that works by default takes many seconds to read barcode correctly

- Before
image
image

- After
image
image

### CODE REVIEW

File: BlazorBarcodeScanner.ZXing.JS/BarcodeReader.razor.cs [hunk 1]
```diff
@@ -128,6 +152,12 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
 
                     if (StartCameraAutomatically && _videoInputDevices.Count > 0)
                     {
+                        if (_videoInputDevices.Count > 0
```
Comment (sabitertan) [inline BlazorBarcodeScanner.ZXing.JS/BarcodeReader.razor.cs:155]: This is pretty specific use case, it is up to user to decide, can't merge this.

File: BlazorZXingJSApp/Client/Pages/Index.razor [hunk 2]
```diff
@@ -10,8 +10,8 @@
             @ref="_reader"
             StartCameraAutomatically="true"
             Title="Scanzor"
-            VideoWidth="720"
-            VideoHeight="540"
+            VideoWidth="400"
```
Comment (sabitertan) [inline BlazorZXingJSApp/Client/Pages/Index.razor:13]: Demo should stay with higher resolution.

File: BlazorBarcodeScanner.ZXing.JS/BarcodeReader.razor.cs [hunk 3]
```diff
@@ -9,6 +9,27 @@ namespace BlazorBarcodeScanner.ZXing.JS
 {
     public partial class BarcodeReader : ComponentBase, IDisposable, IAsyncDisposable
     {
+        [Parameter]
+        public int SelectStartCameraAutomatically { get; set; } = 1;
```
Comment (sabitertan) [inline BlazorBarcodeScanner.ZXing.JS/BarcodeReader.razor.cs:13]: Too specific, remove please.

File: BlazorBarcodeScanner.ZXing.JS/BarcodeReader.razor [hunk 4]
```diff
@@ -12,20 +12,32 @@
...
 
-    @if (ShowStart || ShowReset || ShowToggleTorch)
+    @if (ShowStart || ShowReset || ShowToggleTorch || ShowStop)
     {
         <div>
             @if (ShowStart)
             {
-                <button class="zxing-button start" @onclick="StartDecodingSafe">Start</button>
+                <button class="zxing-button start" @onclick="StartDecodingSafe">
+                    @ButtonStartText
```
Comment (sabitertan) [inline BlazorBarcodeScanner.ZXing.JS/BarcodeReader.razor:21]: Good Idea for supporting multilanguage, thanks!

**Comment by programadormtg6** [comment]
- Requested changes were made

New features:

- Change button texts.

- Change device selection label text.

- Change text when camera does not start automatically

**Comment by dframirez80** [comment]
- Requested changes were made

New features:

- Change button texts.

- Change device selection label text.

- Change text when camera does not start automatically

**Review by sabitertan — APPROVED** [review]

**Pull request merged**